tests/test_macos.py

```diff
@@ -1,46 +1,31 @@
 from __future__ import annotations
 
 import os
-import sys
 from typing import Any
 
 import pytest
 
 from platformdirs.macos import MacOS
 
 
-@pytest.mark.skipif(not sys.platform.startswith("darwin"), reason="tests are platform-specific to macOS")
 @pytest.mark.parametrize(
     "params",
     [
-        {},
-        {"appname": "foo"},
-        {"appname": "foo", "version": "v1.0"},
-    ],
-    ids=[
-        "no_args",
-        "app_name",
-        "app_name_version",
+        pytest.param({}, id="no_args"),
+        pytest.param({"appname": "foo"}, id="app_name"),
+        pytest.param({"appname": "foo", "version": "v1.0"}, id="app_name_version"),
     ],
 )
 def test_macos(params: dict[str, Any], func: str) -> None:
     result = getattr(MacOS(**params), func)
 
     home = os.path.expanduser("~")
-    suffix_elements = []
-    if "appname" in params:
-        suffix_elements.append(params["appname"])
-    if "version" in params:
-        suffix_elements.append(params["version"])
-    if suffix_elements:
-        suffix_elements.insert(0, "")
-    suffix = "/".join(suffix_elements)
```

Comment:
BTW, that was c&p from another test module. So guess you want to "fix" it there, too.